model/telemetry/metrics.yaml

```diff
@@ -0,0 +1,91 @@
+groups:
+  - id: metric.telemetry.sdk.span.ended_count
+    type: metric
+    metric_name: telemetry.sdk.span.ended_count
+    stability: experimental
+    brief: "The number of spans which have been ended"
+    instrument: counter
+    unit: "1"
+
+  - id: metric.telemetry.sdk.span.sampled_count
+    type: metric
+    metric_name: telemetry.sdk.span.sampled_count
+    stability: experimental
+    brief: "The number of spans which have been ended AND are sampled"
+    instrument: counter
+    unit: "1"
+
+  - id: metric.telemetry.sdk.span.processor.queue_size
+    type: metric
+    metric_name: telemetry.sdk.span.processor.queue_size
+    stability: experimental
+    brief: "The number of spans in the queue of a given instance of an SDK span processor"
+    note: |
+      Only applies to span processors which use a queue, e.g. the SDK Batching Span Processor.
+    instrument: updowncounter
+    unit: "1"
+    attributes:
+      - ref: telemetry.sdk.processor.type
+      - ref: telemetry.sdk.component.id
+
+  - id: metric.telemetry.sdk.span.processor.queue_capacity
+    type: metric
+    metric_name: telemetry.sdk.span.processor.queue_capacity
+    stability: experimental
+    brief: "The maximum number of spans the queue of a given instance of an SDK span processor can hold"
+    note: |
+      Only applies to span processors which use a queue, e.g. the SDK Batching Span Processor.
+    instrument: gauge
+    unit: "1"
+    attributes:
+      - ref: telemetry.sdk.processor.type
+      - ref: telemetry.sdk.component.id
+
+  - id: metric.telemetry.sdk.span.processor.spans_submitted
+    type: metric
+    metric_name: telemetry.sdk.span.processor.spans_submitted
+    stability: experimental
+    brief: "The number of spans submitted for processing to this span processor"
+    instrument: counter
+    unit: "1"
+    attributes:
+      - ref: telemetry.sdk.processor.type
+      - ref: telemetry.sdk.component.id
+
+  - id: metric.telemetry.sdk.span.processor.spans_failed
+    type: metric
+    metric_name: telemetry.sdk.span.processor.spans_failed
+    stability: experimental
+    brief: "The number of spans which could not be successfully processed"
+    instrument: counter
+    unit: "1"
+    attributes:
+      - ref: telemetry.sdk.processor.type
+      - ref: telemetry.sdk.component.id
+      - ref: error.type
+        brief: >
+          A low-cardinality description of the failure reason. SDK Batching Span Processors MUST use `queue full` for spans dropped due to a full queue.
+        examples: ["queue full"]
+
+  - id: metric.telemetry.sdk.span.exporter.spans_submitted
+    type: metric
+    metric_name: telemetry.sdk.span.exporter.spans_submitted
+    stability: experimental
+    brief: "The number of spans submitted for export"
+    instrument: counter
+    unit: "1"
+    attributes:
+      - ref: telemetry.sdk.exporter.type
+      - ref: telemetry.sdk.component.id
+
+  - id: metric.telemetry.sdk.span.exporter.spans_failed
```

Comment:
We don't usually define separate metrics for failed/successful - we define a single one and use `error.type` attribute as a marker that something has failed

Reply:
I'll rename the metric to `spans_processed` to include both successful and failed processings with distinguishable based on the presence of `error.type`.